packages/iov-tendermint-rpc/src/responses.ts

```diff
@@ -115,6 +116,22 @@ export interface ValidatorsResponse {
   readonly results: ReadonlyArray<Validator>;
 }
 
+// Events
+
+export interface NewBlockEvent extends Block {}
+
+export interface NewBlockHeaderEvent extends Header {}
+
+export interface TxEvent {
```

Comment:
Hmmm... odd that `TxEvent` is different than `TxResponse` (subscribe vs. query).

I mean, I guess proof would always be empty here, but I would assume `txResult` could be present in `TxEvent` and `result` with tags present in `TxResponse`.

I should investigate tendermint more.

Reply:
I did not have the experience and the motivation to dig into that. What I saw at a first glance was different structures so I built the type from my example return data. Maybe it can be uniform, maybe not.

Reply:
I think you are right the data is different. They really don't understand the need for a nice rpc api... 18 months later....